CODEOWNERS

```diff
@@ -0,0 +1,88 @@
+/cpu/cc2538/                                @hexluthor @smlng
+/cpu/cc26x0/                                @hexluthor @smlng
+/cpu/cc26x2_cc13x2/                         @hexluthor @smlng
+/cpu/cc26xx_cc13xx/                         @hexluthor @smlng
+/cpu/cortexm_common/                        @haukepetersen @thomaseichinger @DipSwitch
+/cpu/efm32/                                 @basilfx
+/cpu/esp*/                                  @gschorcht
+/cpu/kinetis/                               @fjmolinas
+/cpu/lpc2387/                               @benpicco @maribu
+/cpu/mips*/                                 @kaspar030 @francois-berder
+/cpu/msp430*/                               @OlegHahm
+/cpu/sam0_common/                           @benpicco @dylad @keestux
+/cpu/samd21/                                @dylad @biboc
+/cpu/saml1x/                                @dylad
+/cpu/saml21/                                @dylad
+/cpu/stm32*/                                @aabadie @fjmolinas @DipSwitch @vincent-d
+
+/doc/                                       @aabadie @jia200x
+
+/drivers/at86rf2xx/                         @daniel-k @Hyungsin @jia200x @smlng @miri64
+/drivers/cc110x/                            @maribu
+/drivers/cc1xxx_common/                     @maribu
+/drivers/ccs811/                            @gschorcht
+/drivers/enc28j60/                          @haukepetersen
+/drivers/dcf77/                             @daexel
+/drivers/dose/                              @jue89
+/drivers/itg320x/                           @gschorcht
+/drivers/mrf24j40/                          @bergzand
+/drivers/pca9685/                           @gschorcht
+/drivers/sht3x/                             @gschorcht
+/drivers/si70xx/                            @basilfx
+/drivers/sx127x/                            @aabadie @jia200x
+/drivers/ws281x/                            @maribu
+/drivers/xbee/                              @haukepetersen @miri64
+
+/pkg/cryptoauthlib/                         @Einhornhool @PeterKietzmann
+/pkg/lua/                                   @jcarrano
+/pkg/lwip/                                  @miri64
+/pkg/gecko_sdk/                             @basilfx
+/pkg/micropython/                           @bergzand @kaspar030
+/pkg/openthread/                            @Hyungsin @jia200x
+/pkg/u8g2/                                  @basilfx
+/pkg/ucglib/                                @basilfx
+
+/sys/event/                                 @kaspar030
+/sys/evtimer/                               @kaspar030
+/sys/fmt/                                   @kaspar030
+/sys/net/                                   @miri64 @haukepetersen @PeterKietzmann @cgundogan
+/sys/riotboot/                              @kaspar030
+/sys/tsrb/                                  @kaspar030
+/sys/usb/                                   @bergzand @dylad @aabadie
+/sys/xtimer/                                @kaspar030 @MichelRottleuthner
```

Comment:
```suggestion
/sys/xtimer/                                @Hyungsin @MichelRottleuthner
```

Reply:
@Hyungsin is not a maintainer (but maybe that should change ?)

Reply:
Does @kaspar030 not maintain xtimer anymore?